ingestion-beam/bin/generate-avro-test-resources.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
```

Comment:
Can you document in an extended comment or a module docstring here more about the context of this script and how to invoke it? In particular, it took me a while to realize that the output of this script is checked into the repo, so this would only need to be rerun if the developer was needing to change the test avro schemas.